lib/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Bedrock messages-client-http module.
+ *
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node: true */
+
+'use strict';
+
+var bedrock = require('bedrock');
+var config = bedrock.config;
+var brMessagesClient = require('bedrock-messages-client');
+var validate = require('bedrock-validation').validate;
+
+require('bedrock-express');
+
+var brPassport = require('bedrock-passport');
+var ensureAuthenticated = brPassport.ensureAuthenticated;
+var optionallyAuthenticated = brPassport.optionallyAuthenticated;
+
+require('./config');
+
+// TODO: Add logging info
+var logger = bedrock.loggers.get('app');
+
+bedrock.events.on('bedrock-express.configure.routes', function(app) {
+  var basePath = config['messages-client-http'].basePath;
+  // Get ALL clients.
+  app.get(basePath, ensureAuthenticated, function(req, res, next) {
+      brMessagesClient.getAll(req.user.identity, {}, function(err, clients) {
+        if(err) {
+          return next(err);
+        }
+        res.json(clients);
+      });
+    });
+
+  // The restart-job endpoint is operating on the premise that there can only
+  // be one client polling on an endpoint
+  /**
+   * Restarts message clients on the specified endpoint with
+   * the specified interval.
+   *
+   * @param interval the number of minutes to wait between polling
+   * the endpoint (e.g. 1, 5, or 10)
+   * @param endpoints array of endpoints for clients to be restarted
```

Comment:
It actually looks like I changed the method body without changing the docs -- the endpoint accepts a full message client object and will update it in the database (but it won't insert a new message client if it doesn't exist), so POST seems appropriate here (even though it does have the side-effect of it restarting a message client on update...)